core_data_modules/cleaners/codes/codes.py

```diff
@@ -24,4 +24,5 @@ class Codes(object):
     NOT_REVIEWED = "NR"
     NOT_CODED = "NC"
     WRONG_SCHEME = "WS"
-    NOISE_OTHER_PROJECT= "NOP"
+    NOISE_OTHER_PROJECT = "NOP"  # Deprecated; use NOISE_OTHER_CHANNEL instead.
```

Comment:
Do we have a plan as to what we'll need to do to remove this? Can we set a date for its deletioin?

Reply:
When ICRAF completes. ICRAF is the only project using this control code.

Reply:
> When ICRAF completes.

Which of course means we can't set a date now